packages/bpk-docs/src/pages/InfiniteScrollPage/InfiniteScrollPage.js

```diff
@@ -0,0 +1,169 @@
+import { BpkSpinner, SPINNER_TYPES } from 'bpk-component-spinner';
+
+import infiniteScrollReadme from '../../../../bpk-component-infinite-scroll/readme.md';
+
+import DocsPageBuilder from './../../components/DocsPageBuilder';
+import IntroBlurb from './../../components/neo/IntroBlurb';
+
+const elementsArray = [
+  'element 1',
+  'element 2',
+  'element 3',
+  'element 4',
+  'element 5',
+  'element 6',
+  'element 7',
+  'element 8',
+  'element 9',
+  'element 10',
+  'element 11',
+  'element 12',
+  'element 13',
+  'element 14',
+  'element 15',
+  'element 16',
+  'element 17',
+  'element 18',
+  'element 19',
+  'element 20',
+];
+
+const List = ({ elements }) => (
+  <div id="list">
+    {elements.map(element => (
+      <div key={element} style={{ height: '50px' }}>
+        {element}
+      </div>
+    ))}
+  </div>
+);
+
+List.propTypes = {
+  elements: PropTypes.arrayOf(PropTypes.any).isRequired,
+};
+
+const FixedHeighPanel = ({ children }) => (
+  <div style={{ height: '300px', overflow: 'scroll' }}>{children}</div>
+);
+
+FixedHeighPanel.propTypes = {
+  children: PropTypes.node.isRequired,
+};
+
+const InfiniteList = withInfiniteScroll(List);
+
+const components = [
+  {
+    id: 'infinite',
+    title: 'Infinite string list',
+    blurb: [],
+    examples: [
+      <FixedHeighPanel>
+        <InfiniteList
+          onItemsFetch={(index, nElements) =>
+            new Promise(resolve => {
+              setTimeout(() => {
+                const elements = [];
+                for (let i = 0; i < nElements; i += 1) {
+                  elements.push(`element ${index + i}`);
+                }
+                resolve(elements);
+              }, 10);
+            })
+          }
+        />
+      </FixedHeighPanel>,
+    ],
+  },
+  {
+    id: 'infinite1',
+    title: 'One elements per scroll',
+    blurb: [],
+    examples: [
+      <FixedHeighPanel>
+        <InfiniteList
+          onItemsFetch={onItemsFetch(elementsArray)}
+          elementsPerScroll={1}
+        />
+      </FixedHeighPanel>,
+    ],
+  },
+  {
+    id: 'infiniteStop',
+    title: 'Stoping after some scrolls',
+    blurb: [],
+    examples: [
+      <FixedHeighPanel>
+        <InfiniteList
+          onItemsFetch={onItemsFetch(elementsArray)}
+          elementsPerScroll={5}
+          seeMoreAfter={2}
+          seeMoreComponent={<BpkButton>See More</BpkButton>}
+        />
+      </FixedHeighPanel>,
+    ],
+  },
+  {
+    id: 'infiniteCustomLoading',
+    title: 'Using a custom loadingComponent',
+    blurb: [],
+    examples: [
+      <FixedHeighPanel>
+        <InfiniteList
+          onItemsFetch={onItemsFetch(elementsArray)}
+          elementsPerScroll={5}
+          loadingComponent={<BpkSpinner type={SPINNER_TYPES.primary} />}
+        />
+      </FixedHeighPanel>,
+    ],
+  },
+];
+
+const isNeo = process.env.BPK_NEO;
+
+const InfiniteScrollPage = ({ ...rest }) => (
+  <DocsPageBuilder
+    title="Infinite Scroll"
```

Comment:
Can we change to sentence case i.e. 'Infinite scroll'?

Reply:
done